biz.aQute.bndlib.tests/test/test/MultiReleaseTest.java

```diff
@@ -175,6 +175,35 @@ public void testMultiReleaseAnalysis() throws Exception {
 
 	}
 
+	@Test
+	public void testPlainBuilder() throws Exception {
+		try (Builder builder = new Builder()) {
+			builder.setProperty("-includeresource", """
+				sun_1_8/=compilerversions/src/sun_1_8/, \
+				META-INF/versions/9/jdk_9_0/=compilerversions/src/jdk_9_0/, \
+				META-INF/versions/11/jdk_11_0/=compilerversions/src/jdk_11_0/, \
+				META-INF/versions/17/jdk_17/=compilerversions/src/jdk_17/, \
+				META-INF/versions/19/jdk_19/=compilerversions/src/jdk_19/, \
+				""");
+
+			Jar jar = builder.build();
+			assertThat(builder.check()).isTrue();
+
+			assertThat(jar.getResource("META-INF/versions/9/META-INF/MANIFEST.MF")).isNotNull();
```

Comment:
```suggestion
			assertThat(jar.getResource("META-INF/versions/9/OSGI-INF/MANIFEST.MF")).isNotNull();
```

The path for the supplemental manifest is **OSGi-INF** not **META-INF** see:

https://docs.osgi.org/specification/osgi.core/7.0.0/framework.module.html#framework.module-multireleasejar

Reply:
you're right. I'd missed that Will fix it